src/index.njk

```diff
@@ -48,7 +48,7 @@
     </head>
     <body class="bg-white dark:bg-sii-darkest-blue dark:text-sii-cool-light-gray flex flex-col min-h-screen">
       {% include "partials/_header.njk" %}
-      <main class="m-6 grow">
+      <main class="grow max-w-600 m-auto px-32">
```

Comment:
the left margin isn't decreasing with smaller form factors. i think we need to set the xs sm md padding?